### PR TITLE
The default sandbox for Salvo.

### PR DESCRIPTION
**Pull request by mum4k**
Instance sizes and resources are minimal for now to save costs. These will be adjusted later on.

Also modifying `.gitignore` to ignore Terraform files.

### CODE REVIEW

File: salvo-remote/sandboxes/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+only.
+
+![Diagram 1 - the default sandbox](images/the_default_sandbox.png)
+
+# Sandbox creation
+
+When Salvo executes, the CI pipeline will produce binaries and VM disk images
+for all the components. Once that is done, a new CI job is executed in the
+`salvo-control` agent pool on AZP. This job is then picked up by a Salvo
+control VM that instruments the sandbox creation.
```
Comment (fei-deng) [inline salvo-remote/sandboxes/README.md:36]: Is the salvo control VM the same thing as 'Salvo Controller' in the Diagram?

File: salvo-remote/sandboxes/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Sandboxes for remote Salvo executions
+
+This directory contains definition of sandboxes. A sandbox are all the
```
Comment (fei-deng) [inline salvo-remote/sandboxes/README.md:3]: A sandbox "has"  all the ... ?

**Review by fei-deng — DISMISSED** [review]
LGTM.  btw, Nice doc as always.

File: salvo-remote/sandboxes/README.md (hunk 2 above)
Comment (mum4k) [reply salvo-remote/sandboxes/README.md]: I meant it to be "are", but the sentence did not read well. Reworded.

Done in a4025efbcc8a4fff046b98bd54d718a517c1583f.

File: salvo-remote/sandboxes/README.md (hunk 1 above)
Comment (mum4k) [reply salvo-remote/sandboxes/README.md]: Thank you for pointing this out, adjusted to use common terminology.

Done in a4025efbcc8a4fff046b98bd54d718a517c1583f.

File: salvo-remote/sandboxes/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+# Sandbox instance life-cycle
+
+The Terraform configuration for sandboxes uses shared state bucket deployed in
+S3 on AWS. The sandbox instances that will be deployed or destroyed are
+determined based on the variables passed to Terraform.
+
+Each of the following variables is a list of build IDs. Each list represents
+one sandbox type. Each listed build ID represents an instance of the sandbox
+type that should be deployed. The build IDs must match the AZP build ID of the
```
Comment (gyohuangxin) [inline salvo-remote/sandboxes/README.md:50]: I'm a little bit confused about it, what is "AZP build ID", is it the `salvo-control` agent ID?

File: salvo-remote/sandboxes/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+repository.
+
+Various sandbox types differ in the amount of instances deployed, amount of
+support systems deployed and the topology.
+
+# Available Sandbox types
+
+## The default sandbox
+
+The default contains the bare minimum of components needed to execute Salvo
```
[the remark this answers is not in the document]
Comment (mum4k) [reply salvo-remote/sandboxes/README.md]: Done in bd193a8a279359b6fd7f1e16756b996730998576.

File: salvo-remote/sandboxes/README.md (hunk 3 above)
Comment (mum4k) [reply salvo-remote/sandboxes/README.md]: Thank you for raising this concern, the text wasn't very clear. I have reworded it and added a clear description of what a `build ID` is. Please reopen if some concepts still need more explanation.

Done in bd193a8a279359b6fd7f1e16756b996730998576.

**Review by gyohuangxin — APPROVED** [review]
LGTM, it's much easier to understand, thanks!

**Pull request merged**